src/shared/ui/widget-title/widget-title.model.ts

```diff
@@ -0,0 +1,26 @@
+import { type VariantProps, cva } from 'class-variance-authority';
+import classNames from 'classnames/bind';
+
+import styles from './widget-title.module.scss';
+
+export type WidgetTitleProps = React.HTMLAttributes<HTMLHeadingElement> & VariantProps<typeof widgetTitleVariants>;
+
+const cx = classNames.bind(styles);
+
+export const widgetTitleVariants = cva(cx('title'), {
+  variants: {
+    mods: {
+      lines: cx('lines'),
+      asterisk: cx('asterisk'),
+    },
+    size: {
+      small: cx('small'),
+      medium: cx('medium'),
+      large: cx('large'),
+    },
+  },
+  defaultVariants: {
+    size: 'medium',
+    mods: null,
+  },
+});
```

Comment:
Why do we need this library, what benefits it provides?

Reply:
https://t.me/c/2142251410/3718